core/tests/webdriverio_desktop/explorationTranslationTab.js

```diff
@@ -70,15 +70,15 @@ describe('Exploration translation and voiceover tab', function() {
     });
     await explorationEditorMainTab.moveToState('second');
     await explorationEditorMainTab.setContent(
-      await forms.toRichText('This is second card.'));
+      await forms.toRichText('This is second card.'), true);
```

Comment:
Passing true in order to wait for the state content editor to fade out else this will throw the following error.
https://github.com/oppia/oppia/actions/runs/3183158470/jobs/5190037226